WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -106,11 +108,14 @@ @interface BlogDetailsViewController () <UIActionSheetDelegate, UIAlertViewDeleg
 @property (nonatomic, strong) NSArray *tableSections;
 @property (nonatomic, strong) WPStatsService *statsService;
 @property (nonatomic, strong) BlogService *blogService;
+@property (nonatomic, strong) NSIndexPath *selectedIndexPath;
```

Comment:
We might want to change the naming of `selectedIndexPath` to something like `restorableSelectedIndexPath`, just something to be a bit more descriptive of how the index path is used for restoration and not necessarily for the actual state of the table's selected index path.